src/main/java/com/openlattice/client/serialization/SerializationConstants.java

```diff
@@ -100,6 +100,7 @@
     public static final String ENTITY_ID_GENERATOR              = "entityIdGenerator";
     public static final String ENTITY_KEY_ID                    = "entityKeyId";
     public static final String ENTITY_KEY_IDS                   = "entityKeyIds";
+    public static final String ENTITY_LINKING_FEEDBACK          = "entityLinkingFeedback";
```

Comment:
If we use`ENTITY_LINKING_FEEDBACK`, `LINKING_FEEDBACK` should be deleted

Reply:
done